terraform/modules/auth_service/secrets.tf

```diff
@@ -7,6 +7,16 @@ data "aws_secretsmanager_secret" "saml_certificate" {
   name = "api-saml-certificate-${var.environment}"
 }
 
+# TODO 836 ticket specific release action: manually create these secrets, copying the current values from the API secrets, so they can be referenced here
+# See api/README.md and terraform/test/main.tf in the Delta repository for setup instructions
+data "aws_secretsmanager_secret" "delta_saml_private_key" {
+  name = "delta-saml-private-key-${var.environment}"
+}
+
+data "aws_secretsmanager_secret" "delta_saml_certificate" {
+  name = "delta-saml-certificate-${var.environment}"
+}
```

Comment:
Should add

```hcl
  lifecycle {
    postcondition {
      condition     = self.kms_key_id == aws_kms_key.auth_service.arn
      error_message = "Secret must use the auth service KMS key"
    }
  }
```
to these, similar to a couple of the other secrets in this file, and make sure the secret on test uses the auth service KMS key